src/components/TodoApp.vue

```diff
@@ -165,6 +134,70 @@ function searchToDos(item: string) {
   searchQuery.value = item
 }
 
+function disactivateSort(isActive: boolean) {
+  isSortActive.value = isActive
+}
+
+function sortTodos(property: string, order: string, isActive: boolean) {
+  activeOrder.value = order
+  activeProperty.value = property
+  isSortActive.value = isActive
+  todos.value.sort((a: Todo, b: Todo) => {
+    switch (property) {
+      case 'Title':
+        return sortByTitle(a, b)
+      case 'Description':
+        return sortByDescription(a, b)
+      case 'Priority':
+        return sortByPriority(a, b)
+      case 'Date':
+        return sortByDate(a, b)
+      default:
+        return 0
+    }
+  })
+}
+
+function sortByTitle(a: Todo, b: Todo) {
+  let fa = a.title.trim().toLowerCase(),
+    fb = b.title.trim().toLowerCase()
+  if (activeOrder.value === 'ascending') {
+    return fa < fb ? -1 : fa > fb ? 1 : 0
+  }
+  console.log('desc')
+  return fa < fb ? 1 : fa > fb ? -1 : 0
+}
+
+function sortByDescription(a: Todo, b: Todo) {
+  let fa = a.description.trim().toLowerCase(),
+    fb = b.description.trim().toLowerCase()
+  if (activeOrder.value === 'ascending') {
+    return fa < fb ? -1 : fa > fb ? 1 : 0
+  }
+  return fa < fb ? 1 : fa > fb ? -1 : 0
+}
+
+function sortByDate(a: Todo, b: Todo) {
+  let fa = new Date(a.dueDate)
+  let fb = new Date(b.dueDate)
+  console.log(typeof fa)
+  console.log(typeof fb)
+  if (activeOrder.value === 'ascending') {
```

Comment:
You can use ternary in this case: 
`return activeOrder.value === 'ascending' ? fa - fb : fb - fa`